hadoop-ozone/dist/dev-support/bin/dist-layout-stitching

```diff
@@ -110,7 +110,6 @@ run cp "${ROOT}/hadoop-ozone/common/src/main/bin/stop-ozone.sh" "sbin/"
 run mkdir -p "./share/hadoop/ozoneplugin"
 run cp "${ROOT}/hadoop-ozone/objectstore-service/target/hadoop-ozone-objectstore-service-${HDDS_VERSION}-plugin.jar" "./share/hadoop/ozoneplugin/hadoop-ozone-datanode-plugin-${HDDS_VERSION}.jar"
 
```

Comment:
We can open a new Jira to remove share/hadoop/ozoneplugin also?

Reply:
Yes, we can open one. But rename instead of remove. ozoneplugin is the directory of the ozone datanode plugin for legacy hdfs clusters. We need the jar file but can be moved to share/ozone/datanodeplugin